### PR TITLE
Allow !Lvl command to match any truncated skill name (if unique)

### PR DESCRIPTION
**Pull request by SoyChai**
Follow-up on yesterday's pr. Now relies on startsWith() for skill matching, with a small map for alternate names that can't be matched in this way.

The end result is that `!Lvl sm` will get you `!Lvl Smithing`, etc. Ambiguous strings will just fail.

### CODE REVIEW

**Comment by SoyChai** [comment]
Would you feel like doing a review on this, @deathbeam? This pr seems to be stalled for some reason and I'm not sure if it needs any changes.

File: runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/SkillAbbreviations.java [hunk 1]
```diff
@@ -85,6 +63,41 @@
...
+	 * @return Full skill name if recognized, else the original string.
+	 */
+	private static String getFullNameStartingWith(String abbrev)
+	{
+		List<Skill> matches = new ArrayList<>();
+		for (Skill skill : Skill.class.getEnumConstants())
+		{
+			if (skill.getName().toUpperCase().startsWith(abbrev.toUpperCase()))
+			{
+				matches.add(skill);
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/SkillAbbreviations.java]: probably just do early return here with first match and then at the end just return the original input? Not sure, I guess this logic is here to check for ambiguous matches, but first match is as good imo.

Comment (SoyChai) [reply runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/SkillAbbreviations.java]: Forgot to comment back on this, but the loop's been cleaned up now.

**Comment by SoyChai** [comment]
Closing this pending a rewrite.

**Pull request closed without merging**